scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/GraphQLMerlinDatabaseService.java

```diff
@@ -577,7 +580,7 @@ mutation createAllPlanActivityDirectives($activities: [activity_directive_insert
           .add("plan_id", planId.id())
           .add("type", act.getType().getName())
           .add("start_offset", act.startOffset().toString())
-          .add("anchored_to_start", act.anchoredToStart());
+          .add("anchored_to_start", act.anchorId() == null || act.anchoredToStart());
```

Comment:
This change should be reverted, as an activity can be anchored to the **end** of the plan

Reply:
@JoelCourtney I will revert this, let me know if you think it should stay